pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1293,6 +1299,28 @@ func setJvmDefault(jvm *api.JvmObject) {
 	}
 }
 
+func setKafkaExtDefault(kafka *api.KafkaExt) {
+	if kafka != nil {
+
+		if kafka.BootstrapServers == "" {
+			//if aren't present bootstrap servers we consider a disabled kafka extension
+			kafka.ExtDisabled = constants.KIE_KAFKA_EXT_DISABLED_DEFAULT
```

Comment:
there is a reason to set it here if, at this point, if kafka object exists, this env should be true already?

Reply:
yep, was a wrong consideration about UI, but could be removed